Exercise_A/assets/js/script.js

```diff
@@ -1 +1,12 @@
 // Write your JavaScript script here.
+jQuery(function($){
+	var $navbar   = $('.x-navbar');
+	$(window).scroll(function(event) {
+		var $current = $(this).scrollTop();
+		if( $current > 0 ){
+			$navbar.addClass('x-navbar-fixed-top');
+		} else {
+			$navbar.removeClass('x-navbar-fixed-top');
+		}

```

Comment:
Nice use of the add class function to add that class making it stick
You can probably combine you js files